practice-02/README.md

```diff
@@ -0,0 +1,58 @@
+# JavaScript #Practice-02
+## Overview
+#### INTRODUCE
+- Creates a phone list, which lists the phones by filter and displays the basic information of each phone.
+- Use the manufacturer's filter to display manufacturer's products.
+- Use price filters to display the price segment of each product
+#### DESIGN
+- This is the design of the app : [link](https://drive.google.com/file/d/1XpJmiGw6D0fnW-4didAcc8v_aNNGkc4J/view?usp=drive_link)
+---
+## TIMELINE
+- Start date: 31/05/2023.
+- Estimate time: 11 days.
+- End date :10/06/2023. (Except Saturday and Sunday)
+####  TIME MANAGER
+- Management: [Trello](https://trello.com/b/2cSRPb4Z/javascript-practice-02)
+---
+## TECHNICAL
+- [HTML5](https://developer.mozilla.org/en-US/docs/Web/HTML)/[CSS3](https://developer.mozilla.org/en-US/docs/Web/CSS)
+- [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+- [ES 6](http://es6-features.org/)
+- [JSON](https://www.w3schools.com/js/js_json_intro.asp)
+- [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)
+---
+## REQUIREMENT
+- Displays a list of phones for sale by price filter and manufacturer. Users can also view listings of phones at other price points by desired manufacturers.
+- The app has 3 features:
+    - Show all products without any filter first
+    - Includes filters to help customers choose the product they want
+    - List of phone manufacturers to help users choose
+#### ANALYSIS :
+- This is the design analysis for the application : [link](https://docs.google.com/document/d/1MF8DaLU2YsnOjFPMEMo7l8BASeBk6MHj2dXO-IhWXag/edit#heading=h.u1h91ao55ie5)
+---
+## TARGET
+- Understand & apply knowledge HTML/CSS/JavaScript (with ES6 syntax).
+- Using javascript to build a filter that displays a list of products by manufacturer and price.
+- Learn and apply some JS built-in helpers like map,reduce ,find , sort, filter.

```

Comment:
code style. Seem you are not learning about this. it's not critical but it's quality